DESCRIPTION

```diff
@@ -2,12 +2,12 @@ Package: ggplot2
 Version: 3.4.0.9000
 Title: Create Elegant Data Visualisations Using the Grammar of Graphics
 Authors@R: c(
-    person("Hadley", "Wickham", , "hadley@rstudio.com", role = "aut",
+    person("Hadley", "Wickham", , "hadley@posit.co", role = "aut",
            comment = c(ORCID = "0000-0003-4757-117X")),
     person("Winston", "Chang", role = "aut",
            comment = c(ORCID = "0000-0002-1576-2126")),
     person("Lionel", "Henry", role = "aut"),
-    person("Thomas Lin", "Pedersen", , "thomas.pedersen@rstudio.com", role = c("aut", "cre"),
+    person("Thomas Lin", "Pedersen", , "thomas.pedersen@posit.co", role = c("aut", "cre"),
```

Comment:
Also need to update the RStudio line at the bottom of the authors to "Posit Software, PBC"

Reply:
We will have to wait for spring cleaning with this as it was already submitted